poonam/JS/Assignment6_js/que1.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>que1</title>
+    </head>
+    <body>
+        <h1>Name List</h1>
+        <script>
+        var input=[];
+        for(let i=0; i<=4; i++){
+            input[i]=prompt("Enter name:");
```

Comment:
add proper lable for asking input like enter one by one five names in prompt box.